packages/common/src/projects/_internal/ProjectUiSchemaEdit.ts

```diff
@@ -198,117 +175,216 @@ export const buildUiSchema = async (
               },
             },
           },
-        ],
-      },
-      {
-        type: "Section",
-        labelKey: `${i18nScope}.sections.timeline.label`,
-        elements: [
           {
-            scope: "/properties/view/properties/timeline",
-            type: "Control",
+            type: "Section",
+            labelKey: `${i18nScope}.sections.timeline.label`,
             options: {
-              control: "arcgis-hub-timeline-editor",
-              showTitleAndDescription: false,
+              section: "block",
+              helperText: {
+                labelKey: `${i18nScope}.sections.timeline.helperText`,
+              },
             },
+            elements: [
+              {
+                scope: "/properties/view/properties/timeline",
+                type: "Control",
+                options: {
+                  control: "arcgis-hub-timeline-editor",
+                  showTitleAndDescription: false,
+                },
+              },
+            ],
           },
         ],
       },
       {
         type: "Section",
-        labelKey: `${i18nScope}.sections.featuredContent.label`,
-        options: {
-          helperText: {
-            labelKey: `${i18nScope}.sections.featuredContent.helperText`,
-          },
-        },
+        labelKey: "shared.sections.heroBanner.label",
         elements: [
+          // {
```

Comment:
are you leaving this here for future consideration/refactoring of how we handle hero content?

Reply:
Yes.